README.md

```diff
@@ -284,11 +284,11 @@ redistributed under the terms specified in the [LICENSE](/LICENSE) file.
 ExMachina is maintained and funded by thoughtbot, inc.
 The names and logos for thoughtbot are trademarks of thoughtbot, inc.
 
-We love open source software!
-See [our other projects][community] or
-[hire us][hire] to design, develop, and grow your product.
+We love open source software, Elixir, and Phoenix.
+See [our other projects][elixir-phoenix],
+or [hire us][hire] to design, develop, and grow your product.
```

Comment:
Possible to have some keywords in link text? eg "hire our Elixir Phoenix development team"